### ISSUE TITLE
Error while creating new project

### ISSUE THREAD

**Issue by renaatdemuynck** [harvard-edtech/caccl#26]
Hi, I'm getting this error when running `npm init caccl`. Everything goes well up until the point when the server needs to be installed:

```
The syntax of the command is incorrect.
node:child_process:903
    throw err;
    ^

Error: Command failed: mkdir server/src
    at checkExecSyncError (node:child_process:826:11)
    at execSync (node:child_process:900:15)
    at exec (C:\Users\xxxxxxxx\AppData\Local\npm-cache\_npx\6d57d4f7ad4edc8c\node_modules\create-caccl\lib\index.js:49:41)
    at Object.createCACCL [as default] (C:\Users\xxxxxxxx\AppData\Local\npm-cache\_npx\6d57d4f7ad4edc8c\node_modules\create-caccl\lib\index.js:200:5)
    at Object.<anonymous> (C:\Users\xxxxxxxx\AppData\Local\npm-cache\_npx\6d57d4f7ad4edc8c\node_modules\create-caccl\lib\init.js:15:34)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 59416,
  stdout: null,
  stderr: null
}
npm ERR! code 1
npm ERR! path C:\Users\xxxxxxxx\Projects\CACCL-test\caccl-v1-test
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c create-caccl

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxxxxxxx\AppData\Local\npm-cache\_logs\2023-05-09T17_47_58_659Z-debug.log
```

I'm on Windows 10, running a bash terminal inside VSCode.

node: v16.13.2
npm: v8.1.2

**Comment by gabeabrams** [comment]
Perhaps not super helpful, but a good place to start: what are the permissions for the folder you're working in? Perhaps this issue has to do with the current user not being able to create folders?

I'd try checking the folder permissions and also maybe try running the command with `sudo`.

**Comment by renaatdemuynck** [comment]
Hi, thanks for the quick reply.
I tried the command with `sudo` but I got this:

```
$ sudo npm init caccl
bash: sudo: command not found
```

After some googling, I ran VSCode as administrator and tried again but with the same result

**Comment by renaatdemuynck** [comment]
PS: I forgot to add the first lines of the error: `The syntax of the command is incorrect.`
I updated my original post.

**Comment by renaatdemuynck** [comment]
I figured it out!
While examining the stack trace, I found these lines of code in `create-caccl/lib/index.js`:
```
198    exec('mkdir server');
199    exec('cd server && npm init -y');
200 -> exec('mkdir server/src');
201    exec('cd server && npm i --save-dev cross-env nodemon @types/express @types/express-session');
```

When I change line 200 to:
```
198    exec('mkdir server');
199    exec('cd server && npm init -y');
200 -> exec('cd server && mkdir src');
201    exec('cd server && npm i --save-dev cross-env nodemon @types/express @types/express-session');
```

... it worked and the command finished succesfully!

**Comment by renaatdemuynck** [comment]
`mkdir --parents server/src` (or `mkdir -p server/src`) would probably work too.

**EDIT:**
This also gives an error:
```
The syntax of the command is incorrect.
...
Error: Command failed: mkdir -p server/src
```

**Comment by renaatdemuynck** [comment]
I created a pull request

**Comment by renaatdemuynck** [comment]
Just realized I posted this in the wrong repo 😨

**Comment by gabeabrams** [comment]
@renaatdemuynck thank you for submitting the PR! I appreciate your work on this :)

I've merged and re-published the `create-caccl` project (now v2.1.0) so you can use the `npm init caccl` command in the future. Sometimes, due to caching, you might still get an older version. If you're getting an older version, you could try clear your npm cache or use `npm init caccl@latest`.

Take care! Thank you again.
